js/about/passwords.js

```diff
@@ -23,13 +66,17 @@ class SiteItem extends React.Component {
   }
 
   render () {
-    return <tr className='passwordItem'>
-      <td className='passwordActions'>
-        <span className='passwordAction fa fa-times' title='Remove site'
-          onClick={this.onDelete} />
-      </td>
-      <td className='passwordOrigin'>{this.props.site}</td>
-    </tr>
+    return <PasswordsTr data-test-id='passwordItem'>
+      <ActionsTd>
+        <span className={css(styles.passwordAction)}>
+          <span className='fa fa-times' title='Remove site'
```

Comment:
can we go ahead on this and localize title? change it to `data-l10n-id='removeSite'` and then on `passwords.properties` add 

```
removeSite.title=Remove site
```

Also add `fa fa-times` icon to `global.js` under [`appIcons`](https://github.com/brave/browser-laptop/blob/master/app/renderer/components/styles/global.js#L159) property.